tests/test_validate_pr_review_config.py

```
@@ -143,3 +158,166 @@ def test_missing_add_thread_reply_resolve_claude_code_only(self) -> None:
+        assert any(
+            "summary_required_columns must be a non-empty list of non-empty strings"
+            in e
+            for e in errors
+        )
+
+
+class TestCliPathSafety:
+    """CWE-22 path-traversal guards on the CLI entry point."""
+
+    def _run(self, *argv: str) -> subprocess.CompletedProcess[str]:
+        return subprocess.run(
```

Ruff (0.15.11)

[error] `subprocess` call: check for execution of untrusted input

(S603)